opensrp-chw/build.gradle

```diff
@@ -374,7 +386,7 @@ android {
 }
 
 dependencies {
-    implementation('org.smartregister:opensrp-client-chw-core:2.1.3-SNAPSHOT-KK-8@aar') {
+    implementation('org.smartregister:opensrp-client-chw-core:2.1.3-SNAPSHOT-KK-9-LOCAL-1@aar') {
```

Comment:
You published the KK-9 the other day with these changes, you might want to update here